_rules/html-title-is-descriptive-c4a8a4.md

```diff
@@ -182,7 +182,7 @@ Even though a correct `<title>` element is put in the `<head>` of the document,
 
 #### Inapplicable Example 1
 
-This document has a `<title>` element but is inapplicable since the document element is an SVG `<svg>` element.
+This document has a `<title>` element but is inapplicable since the document element is an SVG `svg` element
```

Comment:
Why removing the final period?

(also, why removing the square brackets around `<svg>` but leaving the ones around `<title>` but I guess these two changes are both for another PR).

Reply:
Can we please look past this and tackle changes like `<title>` -> `title` and `element` to `<element>` in a different PR?

Reply:
I purely want this to be spelling related & not how sentences are formulated.

Reply:
Could you, nonetheless, **not** remove the final period of that sentence which is introducing a new unrelated mistake that will need later correction.
I get it that you don't want to correct unrelated issues, but creating new ones still doesn't make sense.